RDKShell/RDKShell.cpp

```diff
@@ -995,6 +995,11 @@ namespace WPEFramework {
             {
                 gWillDestroyEventWaitTime = atoi(willDestroyWaitTimeValue); 
             }
+
+            m_timer.start(RECONNECTION_INITIAL_DELAY_MILLISECONDS);
+            m_timer.setInterval(RECONNECTION_TIME_IN_MILLISECONDS);
```

Comment:
Also, is system service not getting activated before rdkshell and we need to activate system service from rdkshell?. please clarify

Reply:
I has "autostart" = false. I can't be sure that it's activated by UI of something else.